proto/v1/review_service.proto

```diff
@@ -124,6 +141,15 @@ message Review {
 
   string title = 3;
 
+  // The plan associated with the review.
+  // Can be empty.
+  // If plan is set in CreateReviewRequest, we will build the rollout from the plan and link to the review.
```

Comment:
nit: comment about the format of plan and rollout.

This is OUTPUT_ONLY?

Reply:
>  // If plan is set in CreateReviewRequest, we will build the rollout from the plan and link to the review.

So not output only

Reply:
Added OUTPUT_ONLY